Section 2/ex1/Main.cpp

```diff
@@ -0,0 +1,9 @@
+#include <iostream>
+
+int main() {
+	std::cout << "Welcome to the game!\nWARNING! This game is extremely gory, bloody, and gross. you need to be at least 18 years old to enter.\nPlease enter your age: ";
+	int age;
+	std::cin >> age;
+	std::string response = (age > 17) ? "Good! You can play the game! But I lied... there is no game :(\n" : "Toddlers aren't allowed to play it.\n";
```

Comment:
instead of using the \n in a string you can use std::endl in line 8, so you dont need to write it all the time